sys-apps/tuned/tuned-2.13.0.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1 systemd xdg-utils
+
+DESCRIPTION="Daemon for monitoring and adaptive tuning of system devices"
+HOMEPAGE="https://fedorahosted.org/tuned/"
+SRC_URI="https://github.com/redhat-performance/tuned/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
+	${PYTHON_DEPS}
+	dev-python/configobj[${PYTHON_USEDEP}]
+	dev-python/decorator[${PYTHON_USEDEP}]
+	dev-python/pyudev[${PYTHON_USEDEP}]
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/python-linux-procfs[${PYTHON_USEDEP}]"
+
+DEPEND="
+	${CDEPEND}"
+
+RDEPEND="
+	${CDEPEND}
+	sys-apps/dbus
+	sys-apps/ethtool
+	sys-power/powertop
+	sys-process/procps
```

Comment:
you do not need this dependency

Reply:
Also due to copying. How do you determine this isn't needed? Reading the spec file for dependencies?

Reply:
it is a part of the system set